bitcoin/src/crypto/key.rs

```diff
@@ -13,6 +13,9 @@ use hashes::hex::FromHex;
 use hashes::{hash160, hex, Hash};
 use internals::write_err;
 pub use secp256k1::{self, constants, KeyPair, Parity, Secp256k1, Verification, XOnlyPublicKey};
+#[cfg(feature = "rand-std")]    
+pub use secp256k1::rand::thread_rng;
```

Comment:
```suggestion
pub use secp256k1::rand;
```

Reply:
`clippy` should have warned for unused import on this line. There are githooks to help you catch things like this before hitting CI to save yourself time, see readme for instructions on using githooks.

Reply:
I ran the githooks manually, and all I see is the diff between the commits.

Reply:
There should be no output from running the git hooks, if there is output then there is a problem.